egs/librispeech/s5/local/chain/run_tdnn_6z.sh

```diff
@@ -0,0 +1,208 @@
+set -e
+
+# configs for 'chain'
+affix=
+stage=11
+train_stage=-10
+speed_perturb=true
+get_egs_stage=-10
+dir=exp/chain/tdnn_6z  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+
+# TDNN options
+# this script uses the new tdnn config generator so it needs a final 0 to reflect that the final layer input has no splicing
+# smoothing options
```

Comment:
You can remove the pooling options. I have raised an issue #653 for this. I will be removing these from other recipes too.